code/modules/cm_aliens/structures/special/recovery_node.dm

```diff
@@ -30,4 +30,4 @@
 				SPAN_HELPFUL("You feel a warm aura envelop you."))
 	if(!do_after(picked_candidate, heal_cooldown, INTERRUPT_MOVED, BUSY_ICON_MEDICAL))
 		return
-	picked_candidate.gain_health(heal_amount)
+	picked_candidate.gain_health(heal_amount*picked_candidate.caste.heal_resting)
```

Comment:
If this affected more than just a few castes maybe. But it would need to cut the heal amount probably back down to 10. For this as is it feels weird that only hellhounds, lurkers, runners, and warriors to be getting 1.4-2.5x the healing boost here. Its not like they don't get that bonus in the first place, they'd just be double dipping here on it more.